docs/pages/mydoc/crdts.md

```diff
@@ -0,0 +1,240 @@
+
+Intuitively concurrent updates are resolved by taking the maximum in the case of concurrent `set`-operations and by considering all increments and decrement operations which are not overwritten by the `set`-operation.
+
+More precisely, the result of reading the value can be specified as follows:
+
+- If there are no `set` operations, the semantics are equivalent to the counter datatype.
+- Otherwise, let `S` be the set of all concurrent `set`-operations, which have not been overridden.
+- For each operation `o` in `S` with argument `assign(v)`, take all increments and decrements concurrent to `o` and aggregate them onto `V`.
+- Return the maximum of the values obtained from the previous step.
+
+#### Operations
+
+increment(v)
+:	Increments the integer by `v`.
+
+decrement(v)
+:	Decrements the integer by `v`.
+
+set(v)
+:   Sets the value of the integer to `v`.
+
+reset()
+:   Same as `set(0)`.
+	This is not a standard implementation of `reset` since  it might affect concurrent updates and does not reset the internal state to the initial state.
+
+## Flags
+
+Flags store a Boolean value (`true` or `false`).
+
+Antidote has two variants for flags, which differ in how concurrent updates are resolved.
+
+The **enable-wins flag** (EW-flag) returns true, if (and only if) there is an `enable`-operation among the latest operations.
+
+The **disable-wins** (DW-flag) flag return true, if (and only if) is an `enable`-operation and no `disable` operation among the latest operations.
+
+Here "latest operations" are all operations which have not been overridden by a later operation.
+
+
+#### Operations
+
+enable()
+:	Sets the flag to `true`.
+
+disable()
+:   Sets the flag to `false`
+
+reset()
+:   Resets the flag to the initial state (`false`).
+
+
+## Maps
+
+Maps in Antidote can store values under keys.
+The keys are simple (binary) values.
+Values are again datatypes and updating nested values is done by sending operations to these embedded values.
+Similarly, reading a value reads all the embedded values and returns them with their respective key.
+
+The different variations of maps differ in how they handle `remove`-operations.
+
+All implementations have in common, that they call `reset` on the removed value.
+Intuitively, a reset-operation Should have the effect of removing all previous operations, so that the state is equivalent to the initial state, but operations concurrent to the `reset` are not lost.
+However, not all datatypes support a `reset` and not all follow this principle (usually for performance reasons).
+
+
+
+
+#### Operations
+
+update(nestedOps)
+:	Sends a list of nested operations to the
+
+remove(keys)
+:   Removes entries from the map based on keys.
+
+reset()
+:   Resets the map.
+
+
+### Grow-only Map (G-Map)
+
+The grow-only map does not support removing entries.
+It can be used for representing objects/structs in Antidote, where fields are dynamically added and removed.
```

Comment:
I missed a "not" in that sentence ;)

The idea is to use it for something like a `user` struct with `name`, `mail`, `passwordhash` and other fields, where the set of fields usually does not change.